src/config.rs

```diff
@@ -2358,9 +2358,6 @@ pub struct TiKvConfig {
     #[online_config(hidden)]
     pub panic_when_unexpected_key_or_data: bool,
 
-    #[online_config(skip)]
-    pub enable_io_snoop: bool,
```

Comment:
mark it as deprecated instead of deleting it.
Add 
```
  #[doc(hidden)]
    #[serde(skip_serializing)]
```

Reply:
I added these two lines but still get error: missing field `enable_io_snoop` in initializer

Reply:
yes, you still need to initialize it.